test/install_wkhtmltoimage.sh

```diff
@@ -7,7 +7,8 @@ sudo apt-get -qq update
 
 sudo apt-get install -y xfonts-75dpi
 wget http://download.gna.org/wkhtmltopdf/0.12/0.12.2/wkhtmltox-0.12.2_linux-trusty-amd64.deb
-sudo dpkg -i wkhtmltox-0.12.2_linux-trusty-amd64.deb
+sudo dpkg -i wkhtmltox-0.12.2_linux-trusty-amd64.deb || true
```

Comment:
why was this failing, wouldn't it be better to address the issue instead of letting it fail and ignoring it?